index.html

```diff
@@ -277,13 +277,10 @@
 <p>
 When <code>requestIdleCallback(callback, options)</code> is invoked, the user agent MUST run the following steps:</p>
 <ol>
-<li>
-Let <var>document</var> be the <code>Window</code> object's
-<a href="http://www.w3.org/TR/html5/browsers.html#active-document">active document</a> object. [[!HTML5]]
-</li>
-<li>Increment the <var>document</var>'s <a>idle callback identifier</a> by one.</li>
-<li>Let <var>handle</var> be the <var>document</var>'s <a>idle callback identifier</a>'s current value</li>
-<li>Append <var>callback</var> to <var>document</var>'s <a>list of idle request callbacks</a>, associated with <var>handle</var>.</li>
+<li>Let <var>window</var> be this <code>Window</code> object's <a href="https://www.w3.org/TR/html5/browsers.html#browsing-context">browsing context</a>'s <code>WindowProxy</code> object.</li>
```

Comment:
Agreed; here and below, just stick with "this Window object". No browsing contexts, no WindowProxies.